docs/index.md

```diff
@@ -0,0 +1,136 @@
+---
+homepage: true
+layout: product
+includeInBreadcrumbs: true
+title: GOV.UK Prototype Components
```

Comment:
We’d probably want to follow the examples of other X-GOVUK sites and move some of the content on this page to a ‘Get started’ page, but we can do that in a subsequent PR.

Reply:
Also, perhaps after his is merge, we can update the main `README` to instead link to such a page, save having content be duplicated in 2 places.

Reply:
Ah yes, I forgot to remove the duplicate content from the README. I've changed that now in this branch to just have a link to the docs.

And yeah, possibly that content could get split up into a "Get started" page, with separate pages for contributing and releasing (unless we move that stuff back to the README).